### PR TITLE
add_contact_form

### PR DESCRIPTION
**Pull request by Surafels**
Contact form section

- Add form
- Add inputs

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -170,7 +184,7 @@ <h1 class="contact">Contact me</h1>
      project that needs coding.<br> 
     I'd love to help with it!</p>
     <div class="forms">
-    <form method="post" action="">
+    <form method="post" action="https://formspree.io/f/xvojodzg">
         <input type="text" name="name" id="name"  placeholder="Lorem Ipsum" maxlength="30">
```
Comment (usorfaitheloho) [inline index.html]: Kindly add the `required` attribute to activate the validation on submission of your form 😄

File: styles.css [hunk 2]
```diff
@@ -706,25 +711,161 @@ hr {
...
+    flex-direction: column;
+    float: right;
+    width: 1156px;
+    height: 496px;
+    flex-shrink: 0;
+    border-radius: 24px;
+    border: 1px solid var(--neutrals-n-40, #dfe1e6);
+    background: var(--neutrals-n-0, #fff);
   }
-  */
```
Comment (usorfaitheloho) [inline styles.css:730]: Kindly place your background image in the right position as it is shown in the figma template given 👍 

Your design :

image

Figma design :
image

File: index.html [hunk 3]
```diff
@@ -170,7 +184,7 @@ <h1 class="contact">Contact me</h1>
...
         <input type="email" name="email" id="email" placeholder="loremipsum@gmail.com" required>
         <textarea  name="message" id="message" placeholder="Write your message here..." cols="30" rows="10" maxlength="500" required ></textarea>
```
Comment (usorfaitheloho) [inline index.html:190]: Please change your placeholder font-color to the color given in the figma design :

Figma design :
image

Your design :
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @Surafels  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Your UI looks good ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html
Comment (AbelG101) [inline index.html:1]: - [ ] [OPTIONAL] The previous reviewer forgot to mention this but I noticed that the branch name you used is not very descriptive. To ensure good development practices, we recommend using a branch name that clearly indicates the feature or action you're working on. For instance, `add_contact_form` would be a more descriptive and identifiable branch name. When working with GitHub, it's important to use descriptive branch names to help collaborators understand the context of your changes. Please consider using this approach or a similar one in the future to help streamline your development process and improve collaboration. Thank you! 🙂

Comment (AbelG101) [inline index.html:1]: - [ ] [OPTIONAL] When creating a pull request, it's important to provide a descriptive title that accurately summarizes the changes made. This helps reviewers understand the purpose of the changes and makes it easier to search for and reference the pull request in the future. A good title should include the name of the relevant feature or bug fix, and provide enough detail to be concise and clear. So please make sure to work on this next time. 😉

**Review by AbelG101 — APPROVED** [review]
# Hi @Surafels & @ 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- The UI is looking good ✔
- Form inputs are properly validated  ✔
- Correct use of GitHub Flow ✔
- All linter checks are passing ✔

## Optional suggestions
- Check the comments below...

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**